website/docs/cloud-docs/api-docs/policies.mdx

```diff
@@ -274,14 +347,15 @@ This PATCH endpoint requires a JSON object with the following properties as a re
+| Key path                         | Type           | Default                                       | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| -------------------------------- | -------------- | --------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data.type`                      | string         |                                               | Must be `"policies"`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `data.attributes.name`           | string         | (Current name)                                | Ignored if present.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
really? wait, why is it here then lol

Reply:
Its copy pasted from the original api docs, I guess if it defaults to the current name, we can remove it :D